### DISCUSSION TITLE
How can carla represents occlusions?

### DISCUSSION THREAD

**Discussion by ccconquer** [carla-simulator/carla#7799] [General]
How does CARLA represent occlusion and the resulting blind spots (from the perspective of the ego vehicle, such as the blue car in the image), and how can it correctly obtain environmental information from the ego vehicle's perspective (e.g., which cars are occluded at certain times and thus not visible to the ego vehicle, meaning it cannot receive information about them)?
![9f24f095adac60ceeb0b4f670e156d9](https://github.com/carla-simulator/carla/assets/111716969/379d4595-d2f4-4b8c-8d93-6b74e3a1adb5)
![ad8b8a85f5b9e18fd735be70696d7a9](https://github.com/carla-simulator/carla/assets/111716969/9c9c5f9e-09fd-4fc2-88fd-733ec2a77a2c)

**Comment by hypez66** [comment]
Hi, I wonder if and how you have solved this issue in CARLA, really appreciate it if you can give me a clue!
